src/main/java/org/jabref/model/entry/BibEntryAdapter.java

```diff
@@ -0,0 +1,58 @@
+package org.jabref.model.entry;
+
+import com.google.gson.TypeAdapter;
+import com.google.gson.stream.JsonReader;
+import com.google.gson.stream.JsonWriter;
+import org.jabref.logic.TypedBibEntry;
+import org.jabref.model.database.BibDatabaseMode;
+import org.jabref.model.entry.field.Field;
+import org.jabref.model.entry.field.InternalField;
+import java.io.IOException;
+import java.util.*;
```

Comment:
Please follow the setup-guide and adjust the intellij configuration --> optimize imports.